index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/program.css">
+    <title>Music Extravaganza</title>
+</head>
+<body> 
+      <header>
+        <nav class="nav1">
+          <a class="nav-link" href="#">Logout</a>
+          <a class="nav-link" href="#">My Page</a>
+          <a class="nav-link" href="#">English</a>
+          <a class="nav-link" href="#" aria-label="twitter" ><i id="twitter-icon" class="fa fa-twitter text-white"></i></a>
+          <a class="nav-link" href="#" aria-label="facebook"><i id="facebook-icon" class="fa fa-facebook text-white"></i></a>
+        </nav>
+        <nav class="nav2">
+          <div class="navlogo"><a href="#"><img src="images/Logo.png" alt="logo"></a></div>
```

Comment:
- [ ] As highlighted by the last reviewer. The logo image is still broken. when I checked your images folder. you saved it as `logo.png` instead of the current `Logo.png`. It will be nice to fix this.

![image](https://user-images.githubusercontent.com/51296741/153727149-10e1f77c-876a-4ca8-9b74-c98183b47f9d.png)